lms/envs/common.py

```diff
@@ -362,7 +362,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
 
     # WIP -- will be removed in Ticket #TNL-4750.
-    'ENABLE_TIME_ZONE_PREFERENCE': False,
+    'ENABLE_TIME_ZONE_PREFERENCE': True,
```

Comment:
Weren't we going to remove the feature flag as part of this release?  I thought we already removed it?  If not, we should prioritize that work right after this PR, right?